test/unit/widgets/peoplePicker.test.ts

```diff
@@ -5,28 +5,118 @@ import {
   PeoplePicker,
   GroupPicker,
   Group,
-  GroupBuilder
+  GroupBuilder,
+  Person
 } from '../../../src/widgets/peoplePicker'
+import ns from '../../../src/ns'
 jest.mock('rdflib')
 jest.mock('solid-auth-client')
-// jest.mock('../../../src/store')
+const kb = require('../../../src/store')
+const fetcher = kb.fetcher
 
 const dom = new JSDOM('<!DOCTYPE html><p>Hello world</p>').window.document
 const element = dom.createElement('div')
 
+describe('FindAddressBook', () => {
+  it('exists', () => {
+    expect(new PeoplePicker().findAddressBook).toBeInstanceOf(Function)
+  })
+  it('runs', () => {
+    expect(new PeoplePicker().findAddressBook('typeIndex')).toMatchObject({})
+  })
+
+  it('should call kb.any and spy load when callback is successful ', () => {
+    const spyOnNowOrWhenFetched = jest.spyOn(fetcher, 'nowOrWhenFetched')
+
+    const spyAny = jest
+      .spyOn(kb, 'any')
+      .mockReturnValueOnce('book')
+      .mockReturnValueOnce('book')
+    const spyLoad = jest.spyOn(fetcher, 'load').mockResolvedValue('book')
+    new PeoplePicker().findAddressBook('typeIndex')
+    const callback: any = spyOnNowOrWhenFetched.mock.calls[0][1]
```

Comment:
If you want to callback to follow the signature of findAddressBook, you can consider typing the method and export it so that you could reuse it here (instead of `any`).

Don't need to do it now if you don't think it gives value, just a tip.